.github/scripts/generate_changelog.py

```
@@ -283,17 +321,29 @@ def main():
-    heading = (
-        f"## Release {version_short} - "
-        f"[Extended Support Release]"
-        f"(https://docs.mattermost.com/product-overview/release-policy.html#release-types)"
-    )
     entry = f"{anchor}\n{heading}\n\n**Release day: {release_date}**\n\n"
  
+    # Build the Go Version section content
+    if go_version:
+        go_section = f"### Go Version\n  - Updated to ``{go_version}``."
+    else:
+        go_section = f"### Go Version\n  - Go version is the same as in the previous release."
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)